packages/maskbook/src/plugins/FileService/types.ts

```diff
@@ -1,5 +1,5 @@
 export interface FileInfo {
-    type: 'arweave'
+    type: 'arweave' | 'skynet'
```

Comment:
@septs it seems like you're using this "tagged" key as "provider", it might be a design mistake cause the database is distinguishing different union types by this "type" key.

Reply:
@septs it ought to be designed as

```ts
provider: 'arweave'
type: 'file'
```